### PR TITLE
 #651 Minor doc updates

### PR DESCRIPTION
**Pull request by crankydillo**
Just some super trivial grammar stuff.  However, there are a few changes to code snippets.

- [x] Title includes issue id.
- [x] Description of the change added.
- [x] Commits are squashed.
- [ ] Tests added.
- [x] Documentation added/updated.
- [x] Also please review [CONTRIBUTING.md](https://github.com/paypal/squbs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: docs/httpclient.md [hunk 1]
```diff
@@ -217,7 +218,7 @@ sample {
...
+We often need to have common infrastructure functionality or organizational standards across different clients.  Such infrastructure includes, but is not limited to, logging, metrics collection, request tracing, authentication/authorization, tracking, cookie management, A/B testing, etc.  As squbs promotes separation of concerns, such logic would belong to infrastructure and not client implementation. The [squbs pipeline](pipeline.md) allows infrastructure to provide components installed into a client the without client owner having to worry about such aspects.  Please see [squbs pipeline](pipeline.md) for more details.
```
Comment (akara) [inline docs/httpclient.md]: ...installed into a client **the** without...

That "the" should not be there, right?

File: docs/http-services.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ public class JavaFlowSvc extends AbstractFlowDefinition {
...
 
-This provides access to the `Flow` representation of the Akka Http low-level server-side API. Please refer to the [Akka HTTP low-level API](http://doc.akka.io/docs/akka-http/current/java/http/server-side/low-level-server-side-api.html#streams-and-http), [Akka Streams](http://doc.akka.io/docs/akka/current/java/stream/stream-quickstart.html), and the [Http model](http://doc.akka.io/docs/akka-http/current/java/http/http-model.html#http-model-java) documentation for further information on constructing more sophisticated `Flow`s.
+This provides access to the `Flow` representation of the Akka HTTP low-level server-side API. Please refer to the [Akka HTTP low-level API](http://doc.akka.io/docs/akka-http/current/java/http/server-side/low-level-server-side-api.html#streams-and-http), [Akka Streams](http://doc.akka.io/docs/akka/current/java/stream/stream-quickstart.html), and the [Http model](http://doc.akka.io/docs/akka-http/current/java/http/http-model.html#http-model-java) documentation for further information on constructing more sophisticated `Flow`s.
```
[the remark this answers is not in the document]
Comment (crankydillo) [reply docs/http-services.md]: I have been a little confused by this as well.  Seems like Github diff-highlight code can get a little confused (perhaps it doesn't recognize a pure-casing diff).  Anyhow, this is the change:
image

I see now that I missed it in a few other places (e.g. the `Http model` link text).  I'll change that as well.

File: docs/httpclient.md (hunk 1 above)
Comment (crankydillo) [reply docs/httpclient.md]: Yes, good catch.  I meant to put it on the other side of `without`:)

`...a client without the client owner having to worry...`

I will change/squash/etc.

**Comment by crankydillo** [comment]
I pushed changes to address the comments.

File: docs/httpclient.md [hunk 3]
```diff
@@ -82,14 +81,15 @@ Below is an `HttpRequest` creation example in Scala.  Please see [HTTP Model Sca
...
+  
+val charset = HttpCharsets.`UTF-8`
+val userData = ByteString("abc", charset.nioCharset())
 val authorization = headers.Authorization(BasicHttpCredentials("user", "pass"))
 
 HttpRequest(
   PUT,
   uri = "/user",
-  entity = HttpEntity(`text/plain` withCharset `UTF-8`, userData),
+  entity = HttpEntity(`text/plain` withCharset charset, userData),
```
Comment (anilgursel) [inline docs/httpclient.md]: Going forward, I think we should prefer ``text/plain`.withCharset(charset)` style.

**Pull request merged**